CHANGELOG.md

```diff
@@ -63,11 +63,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Metroid Prime
 
 - Fixed: Parasite Queen permadeath when skipping death cutscene
+- Fixed: Minor issues regarding PAL Skippable Cutscenes
```

Comment:
Like one weird cutscene in Frigate, dw about it.

Reply:
then mention it please :)
Would've been more understandable if it was affecting like almost every other cutscene. but as it's written right now, it's very much giving me "General system stability improvements to enhance the user's experience." vibes.

Reply:
By frigate I mean space frigate btw.

More specifically it's an addition from my version of the Intro Closing cutscene skip, there is a PAL json that deletes the NTSC cinematic skip functions that I added in the normal json because retro already added cinematic skips in those rooms and etc.

What is the change in question that you ask of? It's that when you skip the cutscene there is 1 frame of gameplay visible before the black screen from the worldteleporter shows up, I changed it so that doesn't happen like my version of the cutscene skip.

Reply:
didnt you also fix HotE and flaahgra tho? i don't see them mentioned in this changelog.

Reply:
Yes, those issues are fixed, I put the Flaahgra fix in my original changelog, but then at last minute you reported the hote issue and I already had made the PR by then, so it's not listed.

Reply:
This is the original changelog btw
https://github.com/toasterparty/randomprime/pull/88

These are way more accurate